requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django>=2.1.3,<2.2.0
```

Comment:
You are using an outdated version of Django. I recommend using the latest LTS if you do not want to use the latest.

Reply:
Changed to last LTS Django version